docs-site/docs/user/organization/groups/bot-accounts/types.md

```diff
@@ -4,4 +4,8 @@ id: types
 title: Bot Account Types
 ---
 
-<!-- TODO: Add description of the types -->
+The different bot account types are:
+
+- **Project bot** - A bot that is created within a project. This bot can only be associated to the project it was created in.
```

Comment:
At present only a `group_bot` type exists for groups so the Project bot and Project automation bot lines can be removed from here

Reply:
30ed197bcd870d4b31e3b30cf22454ef0bee829e